tests/config/webpack/webpack.config.test.js

```diff
@@ -1,68 +1,51 @@
 const webpackConfig = require('../../../config/webpack/webpack.config');
 
+const configExpectations = processPath => ({
+  entry: expect.objectContaining({
+    'terra-dev-site': `${processPath}/lib/Index`,
+    rewriteHistory: `${processPath}/lib/rewriteHistory`,
+    redirect: `${processPath}/lib/redirect`,
+  }),
+  resolve: expect.objectContaining({
+    modules: expect.arrayContaining([
+      `${processPath}/dev-site-config`,
+    ]),
+    alias: expect.objectContaining({
+      'terra-dev-site': `${processPath}`,
+    }),
+  }),
+  plugins: expect.arrayContaining([
+    expect.objectContaining({
+      options: expect.objectContaining({
+        title: 'Terra Dev Site',
+        template: `${processPath}/lib/index.html`,
+        lang: 'en',
+        dir: 'ltr',
+        favicon: `${processPath}/terra-favicon/32px/favicon.ico`,
```

Comment:
Lets add `inject: false` and `headChunk: ['rewriteHistory']` and `rootElementId` to this test

Reply:
We should ensure the `TERRA_DEV_SITE_BASENAME` is defined correctly when the public-output-path changes